scripts/build-flutter-app.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Bash "strict mode", to help catch problems and bugs in the shell
+# script. Every bash script you write should include this. See
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/ for
+# details.
+set -euo pipefail
```

Comment:
I'm getting locally an error locally due to this line:
```
./scripts/build-flutter-app.sh: 7: set: Illegal option -o pipefail
``` 
I'm not sure why it is failing, but I just commented that line and it worked fine (in case the same happens on the pod)